mne/viz/topomap.py

```diff
@@ -15,13 +16,14 @@
 import warnings
 
 import numpy as np
+from scipy.spatial.distance import pdist, squareform
```

Comment:
This will probably need to be a nested import

https://travis-ci.org/mne-tools/mne-python/jobs/660587770#L3259